templates/errors.rb

```diff
@@ -23,7 +23,7 @@
   Net::SMTPAuthenticationError,
   Net::SMTPServerBusy,
   Net::SMTPUnknownError,
-  TimeoutError
+  Timeout::Error
```

Comment:
I agree since the style guide enforces this the other lines are wrong and should be updated. I will add that change to this PR.